### PR TITLE
Helen/nh group references

### PR DESCRIPTION
**Pull request by HacksForHugs**
- Can visit neighborhood -> show method
- Neighborhood show method lists relevant groups as index
- Create new group creates a group under relevant neighborhood

Fix: #24 #39

### CODE REVIEW

**Comment by HacksForHugs** [comment]
@lilshim what does failing the check mean?

**Comment by HacksForHugs** [comment]
@lilshim merged!

File: app/views/users/confirmations/new.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
   <div class="field">
     <%= f.label :email %><br />
```
Comment (hidesminimally) [inline app/views/users/confirmations/new.html.erb:7]: @HacksForHugs why did you add this?

File: config/routes.rb [hunk 2]
```diff
@@ -1,33 +1,13 @@
...
 
-  devise_for :users, :path => '', :path_names => {:sign_up => 'register', :sign_in => 'login', :sign_out => 'logout'}, :controllers => { :registrations => 'users/registrations' }
 
-  get 'static_pages/help'
+  resources :neighborhoods, only: [:index, :show, :new, :create] do
+    resources :groups, only: [:new, :show, :create]
+  end
 
-  get 'static_pages/about'
+  devise_for :users
```
Comment (hidesminimally) [inline config/routes.rb]: @HacksForHugs delete this. you need 

```
devise_for :users, :path => '', :path_names => {:sign_up => 'register', :sign_in => 'login', :sign_out => 'logout'}, :controllers => { :registrations => 'users/registrations' }
```

File: config/routes.rb [hunk 3]
```diff
@@ -1,33 +1,13 @@
 Rails.application.routes.draw do
 
-  root 'static_pages#home'
```
Comment (hidesminimally) [inline config/routes.rb]: @HacksForHugs you need `root 'static_pages#home`. i think you may have deleted too much from your merge conflicts....

File: config/routes.rb [hunk 4]
```diff
@@ -1,33 +1,13 @@
...
 
-  get 'static_pages/help'
+  resources :neighborhoods, only: [:index, :show, :new, :create] do
+    resources :groups, only: [:new, :show, :create]
+  end
 
-  get 'static_pages/about'
+  devise_for :users
 
-  get 'user/show'
```
Comment (hidesminimally) [inline config/routes.rb:11]: @HacksForHugs This needs to be here no?

File: config/routes.rb [hunk 5]
```diff
@@ -1,33 +1,13 @@
...
-
-  get 'user/:id', to: "user#show", as: 'user'
```
Comment (hidesminimally) [inline config/routes.rb:13]: this too

File: app/controllers/neighborhoods_controller.rb [hunk 6]
```diff
@@ -3,30 +3,27 @@ def index
...
-		   else
-		      render 'new'
-		   end
+			redirect_to action: "index"
+		else
+			render 'new'
 		end
-	end
+	end 
+
```
Comment (hidesminimally) [inline app/controllers/neighborhoods_controller.rb]: @HacksForHugs extra space! make sure your precommit is on bruh

File: app/views/users/confirmations/new.html.erb (hunk 1 above)
Comment (HacksForHugs) [inline app/views/users/confirmations/new.html.erb:7]: @lilshim I think this was a result of fixing a merge conflict; I replaced the HEAD>>>> stuff with the MASTER>>>> stuff because I thought that MASTER>>>> stuff was the most up-to-date. I never actually meant to edit this file.

Comment (hidesminimally) [inline app/views/users/confirmations/new.html.erb:7]: aiight @HacksForHugs gotcha. let's keep it for now, and then remove when we refactor

**Comment by HacksForHugs** [comment]
@lilshim
